_data/cores.yml

```
@@ -1,220 +1,252 @@
 ---
 - username: agg23
   cores:
-    - repo: analogue-arduboy
+    - repository: analogue-arduboy
       display_name: Arduboy for Analogue Pocket
       identifier: agg23.Arduboy
       platform: Arduboy
       assets:
-        location: Assets/arduboy/common/
-    - repo: analogue-pong
+      - platform: arduboy
+        extensions:
+        - hex
+    - repository: analogue-pong
       display_name: Atari's 1972 Pong
       identifier: agg23.Pong
       platform: Pong
-    - repo: openfpga-NES
+      # Do we need assets at all if no additional files are required?
```

I think an empty array would be best here.

```suggestion
      assets: []
```